### PR TITLE
Vuetify version upgrade

### PR DESCRIPTION
**Pull request by leksmall**
*Issue #:* bcgov/entity#1258

*Description of changes:*
Changed  vuetify version to 2.0.11 or higher
Changed all UI components to use sass instead of stylus
Replaced the deprecated UI components/properties with the new components/properties

Links for reference
https://stackoverflow.com/questions/55877576/how-to-use-vuetify2-0-0-alpha-17
https://github.com/vuetifyjs/vuetify/releases/tag/v2.0.0

Note : sass loader version 8.0.0 has some issues. Hence going with the stable version 7.2.0
https://github.com/vuejs/vue-cli/issues/4526

Todo
1. Update the version of sbc-common to 2.0.1 once that is published
2.Remove stylus dependency from package.json

*API Code Review Checklist:*
 - #### Common
    - [ ] Proper naming conventions followed on variables, classes, definitions etc.
    - [ ] Proper logging levels are used in the code 
 - #### REST 
    - [ ] Endpoints follows agreed pattern and organized around resources
    - [ ] Conform to http semantics
    - [ ] Proper filtering and pagination is provided for large amounts of data
 - #### Code structure
    - [ ] Resources is used for request payload processing, validations, response codes, messages etc.
    - [ ] Service is used for business logic, 3rd party communication management. Maintains proper folder structure.
    - [ ] Model is used for database models.
    - [ ] Schemas is used for json schema
    - [ ] Util contains commonly used utility functions, constants
 - #### Test Cases
    - [ ] Unit test cases with happy and sad path for service, model (?) layers
    - [ ] Endpoints test cases with happy and sad path for resources
    - [ ] Mocking of services (mainly 3rd party dependencies) provided if applicable
    - [ ] Postman collection tests provided for endpoints with happy path
 - #### Code Quality
    - [ ] Proper code coverage for all the layers (percentage 80, 90 ?)
    - [ ] Linting
- #### DB Migrations
    - [ ] Alembic is used to manage database migration. [Reference](https://github.com/bcgov/namex/blob/master/docs/database.md)

*UI Code Review Checklist:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by sutherlanda** [comment]
Once this PR is merged into post-mvp, we should also merge into development (post-mvp and development are now synced).

File: auth-web/src/App.vue [hunk 1]
```diff
@@ -49,27 +49,32 @@ export default Vue.extend({
 
 </script>
 
-<style lang="stylus">
-  @import "./assets/styl/base.styl";
-  @import "./assets/styl/layout.styl";
-  @import "./assets/styl/overrides.styl";
+<style lang="scss">
+  @import "./assets/scss/base.scss";
```
Comment (pksumesh) [inline auth-web/src/App.vue]: Is stylus to scss needed for this upgrade? Please add any reference so that we will have some reference in future.

**Comment by pksumesh** [comment]
And this has got conflicts

**Comment by sutherlanda** [comment]
> And this has got conflicts

Conflicts are from me merging development into post-mvp.  I can fix up before we merge this one.

**Pull request merged**